src/components/ClickableStyle/ClickableStyle.module.css

```diff
@@ -228,51 +229,52 @@
 }
 
 /**
- * Small button
+ * Small clickable style
  * 1) Height is used instead of vertical padding because the icon size we use would
- *    cause the button height to expand
+ *    cause the height to expand
  */
-.button--sm {
+.clickable-style--sm {
   height: var(--eds-size-3); /* 1 */
   padding-left: var(--eds-size-half);
   padding-right: var(--eds-size-half);
 }
 
 /**
- * Medium button
+ * Medium clickable style
  * 1) Height is used instead of vertical padding because the icon size we use would
- *    cause the button height to expand
+ *    cause the height to expand
  */
-.button--md {
+.clickable-style--md {
   height: var(--eds-size-4); /* 1 */
   padding-left: var(--eds-size-1);
   padding-right: var(--eds-size-1);
 }
 
 /**
- * Large button
+ * Large clickable style
  * 1) Height is used instead of vertical padding because the icon size we use would
- *    cause the button height to expand
+ *    cause the height to expand
  */
-.button--lg {
+.clickable-style--lg {
   height: var(--eds-size-5); /* 1 */
   padding-left: var(--eds-size-2);
   padding-right: var(--eds-size-2);
 }
 
 /**
- * Full-width button
- * 1) A block button fills 100% of the width of its container
+ * Full-width clickable style
+ * 1) A block clickable style that fills 100% of the width of its container
  */
-.button--full-width {
+.clickable-style--full-width {
   width: 100%;
 }
 
 /**
  * Loading icon rotiation style
  * 1) Set SVG icon fill to the current text color
  */
-.button__icon-loading {
+.eds-is-loading svg {
```

Comment:
This was the original name for this class in the Big Medium component. It got changed around when I removed the `icon` style, so here I'm just putting it back the how it should have stayed.